README.md

```diff
@@ -1,17 +1,16 @@
 # gluster-kubernetes
 
-## Dynamic, persistent storage for Kubernetes with GlusterFS
+## GlusterFS + heketi on Kubernetes
```

Comment:
What I am missing in the proposed headline is the *purpuse* of this project. But we should not ly either way. I'll merge and we can then still discuss about a better headline........... :-)